osu.Game/Graphics/Containers/LogoFacadeContainer.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.MathUtils;
+using osu.Game.Screens.Menu;
+using osuTK;
+
+namespace osu.Game.Graphics.Containers
+{
+    /// <summary>
+    /// A container that creates a <see cref="Facade"/> to be used to update and track the position of an <see cref="OsuLogo"/>.
```

Comment:
"update and track"? i'd argue these are opposites.

how about "A container that handles tracking of an `OsuLogo` through different layout scenarios"?

Also rename this to `LogoTrackingContainer` i think? This has far more logic than just the facade.